kie-wb-smoke-tests/src/test/java/org/kie/smoke/wb/AbstractWorkbenchIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
 
     static {
         try {
-            deploymentUrl = new URL(System.getProperty("deployable.base.uri"));
+            deploymentUrl = new URL(System.getProperty("deployable.base.uri", "http://localhost:8080/kie-wb/"));
```

Comment:
This is exactly what I have been doing locally when running the tests from IDE. However, this will only work for kie-wb as kie-drools-wb has different app context. 

To support both I see two options here:
 * make the URL detection kind of smart: try the /kie-wb and if that is not respoding, try /kie-drools-wb)
 * use the same app context for both kie-wb and kie-drools-wb (e.g. just "wb")

I would vote for 2nd option as is seems less tricky.